modules/db/src/main/scala/gem/dao/ObservationDao.scala

```diff
@@ -21,6 +21,7 @@ object ObservationDao {
                     ${o.id.index},
                     ${o.title},
                     ${o.instrument})
+       ON CONFLICT DO NOTHING
```

Comment:
I guess this is debatable. If you want to have the latest snapshot from ocs2 you probably want to delete what you have and overwrite

But yes it can be dangerous